english/33hours/explore-room/room.md

```diff
@@ -1,6 +1,8 @@
 You're not really sure what you're looking for, but search the room frantically to find out where you are and why you
-are here. The nightstand is completely empty. There's not even a phonebook.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.